lib/mustache-js-rails/version.rb

```diff
@@ -1,3 +1,3 @@
 module MustacheJsRails
-  VERSION = "2.3.0"
+  VERSION = "2.3.0.1"
```

Comment:
Shouldn't this be `2.3.1`?

Reply:
It follows `mustache.js` version, which is at `2.3.0` currently. I figured if I used `2.3.1` then it could mislead that `mustache.js` is also `2.3.1`.